lib/core/presentation/widgets/common/photos_gallery_view/photo_gallery_view.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:app/theme/color.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:smooth_page_indicator/smooth_page_indicator.dart';
+
+final IMAGE_SIZE = 351.w;
+
+class PhotoGalleryView extends StatefulWidget {
```

Comment:
Why dont we just put in collaborator widgets ? with prefix ? If you wanna have it in common, please consider use different name, we already have `PhotoGalleryPage`, this causes confusing